### DISCUSSION TITLE
external heater no data

### DISCUSSION THREAD

**Discussion by DillingerAlexander** [kodebach/hacs-idm-heatpump#139] [Q&A]
I have connected an external stove that is displayed as an external heat generator. Whether this is a wood-burning stove or a solar thermal system remains the same.

Unfortunately, this data is not available in the HA

![image](https://github.com/user-attachments/assets/34afcec2-45ed-48ba-9a97-6c5679ffea8c)

**Comment by kodebach** [comment]
How is this external system hooked up? I have a similar situation. In my case it is configured as a separate "Heizkreis" and the values show up as "Raumtemperatur" and "Vorlauftemperatur".

If you are unsure about your configuration, please ask your installer or IDM support for help. After "Fachmann Codeeingabe" in  "Gebäudeleittechnik -> GLT Monitor" you should see all values the heat pump exposes via Modbus. If you find the values there, please post the IDs. With those IDs I can tell you what they show up as in the integration.

**Comment by DillingerAlexander** [comment]
Hi,

I do not see it as a single heating circuit. I'll have to ask my heating engineer if he can take a look at it. Unfortunately, I don't have the specialist code. But I have seen under Information -> Inputs/Outputs -> Sensor inputs. It really seems to be something separate
![image](https://github.com/user-attachments/assets/8235ffa3-09b4-4e31-ab95-e5b60ce4a5cb)
